com/metacube/SCF/Assignment2/HexCalc.java

```diff
@@ -0,0 +1,154 @@
+package GET2018.com.metacube.SCF.Assignment2;
+
+public class HexCalc {
```

Comment:
Add comment at the top of the class specifying purpose, author and copyrights info. Class name should be more meaningful.